templates/components/itilobject/timeline/main_description.html.twig

```diff
@@ -13,10 +13,10 @@
                <div class="d-flex timeline-header">
                   <div class="d-flex creator">
                      {{ include('components/itilobject/timeline/timeline_item_header_badges.html.twig', {
```

Comment:
I guess `anonym_user` variable should be explicitely passed here, as `timeline_item_header_badges.html.twig` is using it.